src/styles/move.scss

```diff
@@ -0,0 +1,55 @@
+.move {
+
+  &__title {
+    font-size: 32px;
+    line-height: 32px;
+    text-align: left;
+    margin-top: 60px;
+    margin-bottom: 32px;
+  }
+
+  &__text {
+    font-size: 23px;
+    line-height: 32px;
+    text-align: left;
+  }
+
+}
+
+
+@media (min-width: 744px) {
```

Comment:
Create mixins for all view variant ( phone, tablet), and use it for specific selector ( not for several)